ext/oci8/tests/password_new.phpt

```diff
@@ -2,6 +2,7 @@
 oci_password_change()
 --SKIPIF--
 <?php
+require 'skipif.inc';
```

Comment:
Looking at the skipif.inc file, this does not look right to me. At the least this would have to be moved until after the `$target_dbs` assignment. I think maybe instead the connect.inc include should be changed to a skipif.inc include, as that's what many other places do.

Though unless you have an environment setup that allows you to run oci8 tests (Travis will not run them), it's probably best to not touch this.

Reply:
@nikic I'll revert this one so, and open a PR only with this changes. I'll need to setup my environment in fact.